server/autolinkclient/client.go

```diff
@@ -64,3 +82,47 @@ func (c *Client) Add(links ...autolink.Autolink) error {
 
 	return nil
 }
+
+func (c *Client) Get(autolinkName string) ([]autolink.Autolink, error) {
```

Comment:
```suggestion
func (c *Client) Get(autolinkName string) ([]*autolink.Autolink, error) {
```
Is there any reason for not using pointer array?